game/src/core/utils/components/draw/Animation.java

```diff
@@ -44,12 +64,29 @@ public Animation(Collection<String> animationFrames, int frameTime, boolean loop
      * Creates an animation. repeats forever
      *
      * @param animationFrames The list of textures that builds the animation. Must be in order.
-     * @param frameTime How many frames to wait, before switching to the next texture?
+     * @param frameTime       How many frames to wait, before switching to the next texture?
      */
     public Animation(Collection<String> animationFrames, int frameTime) {
         this(animationFrames, frameTime, true);
     }
 
+    /**
+     * Create an animation from the files in the given path.
+     *
+     * @param subDir    Path to the subdirectory where the animation frames are stored
+     * @param frameTime Time between two animation frames
+     * @param looping   Should the animation loop or stop at the last frame?
+     * @return The created Animation instance
+     */
+    public static Animation of(File subDir, int frameTime, boolean looping) {
```

Comment:
Siehe Kommentar in `DrawComponent`: Wenn wir zusätzlich noch ein `public static Animation of(File subDir)` anbieten und darin diese Methode hier mit passenden Konstanten aufrufen würden, dann könnten wir den Code in `DrawComponent` deutlich lesbarer gestalten. 

Außerdem gehören die beiden Konstanten `int DEFAULT_FRAME_TIME = 3` und `boolean DEFAULT_IS_LOOP = true` eher hier in diese Klasse und nicht nach `DrawComponent`.